js/ui/camera.js

```diff
@@ -411,7 +413,8 @@ class Camera extends Evented {
      * details not specified in `options`.
      *
      * @memberof Map#
-     * @param {CameraOptions|AnimationOptions} options Options describing the destination and animation of the transition.
+     * @param {Object} options Options describing the destination and animation of the transition.
+    *            Accepts [CameraOptions](#CameraOptions) and [AnimationOptions](#AnimationOptions).
```

Comment:
Should we stick with `{CameraOptions|AnimationOptions}` as the type for this parameter?